zebra/zebra_dplane.c

```diff
@@ -4605,18 +4605,6 @@ enum zebra_dplane_result dplane_intf_update(const struct interface *ifp)
 	return ret;
 }
 
-/*
- * Enqueue a interface delete for the dataplane.
- */
-enum zebra_dplane_result dplane_intf_delete(const struct interface *ifp)
```

Comment:
hmm - why remove this? we might want to allow more plugin-oriented interface management in the future?